### PR TITLE
Add Bitci Chain

### PR DESCRIPTION
**Pull request by ersanyakit**
Add Bitci Chain

## Description

_Concise description of proposed changes_

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address:0xd37af52c1d214027582EAC330AB9854D9F62f8cB

### CODE REVIEW

File: packages/chains/src/bitci.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    blockExplorers: {
+        default: {
+            name: 'BITCI Explorer',
+            url: 'https://bitciexplorer.com/',
+        },
+    },
+    contracts: {
+        multicall3: {
+            address: '0x72dC4C76372c746eB974F97aD70EEB974a7382f4',
```
Comment (jxom) [inline packages/chains/src/bitci.ts:24]: This multicall address isn't verified. Can we either get it verified or remove it?

Comment (ersanyakit) [reply packages/chains/src/bitci.ts:24]: Oh i forget it. I will verify it. thank you very much

Comment (ersanyakit) [reply packages/chains/src/bitci.ts:24]: Bitci MainNet  Multicall Contract
https://bitciexplorer.com/address/0x72dC4C76372c746eB974F97aD70EEB974a7382f4/contracts#address-tabs

Bitci Testnet Multicall Contract
https://testnet.bitciexplorer.com/address/0x2c7a2166C8d89c03D8c04640fD93560FD19fcAF4/contracts

**Review by ersanyakit — COMMENTED** [review]
I fixed.

**Comment by tmm** [comment]
Chains should now be edited/added on the [Viem repository](https://github.com/wagmi-dev/viem).

**Pull request closed without merging**